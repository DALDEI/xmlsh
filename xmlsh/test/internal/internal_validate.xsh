# simple test of validate
for i in *.xsh ; do
  validate $i &&  echo Succeeded validation $i || echo Failed validation $i 
done
echo This should fail
validate ../../samples/scripts/invalid1.xsh 2>/dev/null && \
   echo Failed validation did not detect invalid script || \
   echo Success - validate detected invalid script 