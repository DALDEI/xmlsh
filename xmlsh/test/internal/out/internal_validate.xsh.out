Succeeded validation internal_bool.xsh
Succeeded validation internal_csv2xml.xsh
Succeeded validation internal_csv2xml2.xsh
Succeeded validation internal_csv2xml3.xsh
Succeeded validation internal_csv2xml4.xsh
Succeeded validation internal_fixed2xml.xsh
Succeeded validation internal_null.xsh
Succeeded validation internal_qname.xsh
Succeeded validation internal_quote.xsh
Succeeded validation internal_rngvalidate.xsh
Succeeded validation internal_schematron.xsh
Succeeded validation internal_string.xsh
Succeeded validation internal_time.xsh
Succeeded validation internal_validate.xsh
Succeeded validation internal_xaddbase.xsh
Succeeded validation internal_xbase.xsh
Succeeded validation internal_xcat.xsh
Succeeded validation internal_xcmp.xsh
Succeeded validation internal_xdelattribute.xsh
Succeeded validation internal_xdelattribute2.xsh
Succeeded validation internal_xed.xsh
Succeeded validation internal_xfile.xsh
Succeeded validation internal_xgetopts.xsh
Succeeded validation internal_xgetopts2.xsh
Succeeded validation internal_xgrep1.xsh
Succeeded validation internal_xgrep2.xsh
Succeeded validation internal_xgrep3.xsh
Succeeded validation internal_xidentity.xsh
Succeeded validation internal_xinclude.xsh
Succeeded validation internal_xls.xsh
Succeeded validation internal_xmd5sum.xsh
Succeeded validation internal_xmd5sum2.xsh
Succeeded validation internal_xml2csv.xsh
Succeeded validation internal_xml2csv2.xsh
Succeeded validation internal_xmove.xsh
Succeeded validation internal_xpath.xsh
Succeeded validation internal_xproperties.xsh
Succeeded validation internal_xpwd.xsh
Succeeded validation internal_xquery.xsh
Succeeded validation internal_xquote.xsh
Succeeded validation internal_xsdvalidate.xsh
Succeeded validation internal_xslt.xsh
Succeeded validation internal_xslt1.xsh
Succeeded validation internal_xsplit.xsh
Succeeded validation internal_xsplit2.xsh
Succeeded validation internal_xsplit3.xsh
Succeeded validation internal_xtee.xsh
Succeeded validation internal_xunquote.xsh
Succeeded validation internal_xuri.xsh
Succeeded validation internal_xurlencode.xsh
Succeeded validation internal_xvalidate.xsh
This should fail
Success - validate detected invalid script
